examples/webpack-example/src/app/Main.jsx

```diff
@@ -55,25 +55,27 @@ class Main extends React.Component {
     );
 
     return (
-      <div style={styles.container}>
-        <Dialog
-          open={this.state.open}
-          title="Super Secret Password"
-          actions={standardActions}
-          onRequestClose={this.handleRequestClose}
-        >
-          1-2-3-4-5
-        </Dialog>
-        <h1>material-ui</h1>
-        <h2>example project</h2>
-        <RaisedButton
-          label="Super Secret Password"
-          primary={true}
-          onTouchTap={this.handleTouchTap}
-        />
-      </div>
+      <MuiThemeProvider muiTheme={muiTheme}>
```

Comment:
the muiTheme not calculated correctly. It should be:
```js

const muiTheme = getMuiTheme({
  palette: {
    accent1Color: Colors.deepOrange500,
  },
});
```
Could we use `colors` instead of `Colors`?